src/S3/S3Transfer/Models/S3TransferManagerConfig.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Aws\S3\S3Transfer\Models;
+
+final class S3TransferManagerConfig
+{
+    public const DEFAULT_TARGET_PART_SIZE_BYTES = 8388608; // 8MB
+    public const DEFAULT_MULTIPART_UPLOAD_THRESHOLD_BYTES = 16777216; // 16MB
+    public const DEFAULT_REQUEST_CHECKSUM_CALCULATION = 'when_supported';
+    public const DEFAULT_RESPONSE_CHECKSUM_VALIDATION = 'when_supported';
+    public const DEFAULT_MULTIPART_DOWNLOAD_TYPE = 'part';
+    public const DEFAULT_CONCURRENCY = 5;
+    private const DEFAULT_TRACK_PROGRESS = false;
+    private const DEFAULT_REGION = 'us-east-1';
```

Comment:
Meant to include this in the last round, but we should not be providing default regions or region fallbacks- STS as an example